packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -51,72 +82,130 @@ export default function (plop: NodePlopAPI) {
+          const plopData = data as PlopData;
+          if (plopData.page) {
+            return `Hosted search page "${plopData.page.config.title} has been downloaded`;
+          }
+
+          plopData.page = defaultPageManifest;
+          return 'Using the default search page template.';
+        },
+        {
+          type: 'add',
+          path: currentPath + '/{{project}}/.env',
+          templateFile: '../template/.env.hbs',
+        },
+        {
+          type: 'add',
+          path: currentPath + '/{{project}}/.gitignore',
+          templateFile: '../template/.gitignore.hbs',
+        },
+        {
+          type: 'add',
+          path: currentPath + '/{{project}}/package.json',
+          templateFile: '../template/package.json.hbs',
+        },
+        {
+          type: 'addMany',
+          destination: currentPath + '/{{project}}/',
+          base: '../template',
+          templateFiles: [
+            '../template/src/**',
+            '../template/scripts/*',
+            '../template/.env.example',
+            '../template/tsconfig.json',
+            '../template/stencil.config.ts',
+            '../template/netlify.toml',
+            '../template/README.md',
+            '../template/start-netlify.mjs',
+          ],
+        },
+        function generateTemplates(data) {
+          const {page, project} = data as PlopData;
+          page.results.templates.forEach((resultTemplate, index) => {
+            const filePath = join(
+              currentPath,
+              project,
+              'src',
+              'components',
+              'results-manager',
+              `template-${index + 1}.html`
+            );
+            writeFileSync(
+              filePath,
+              plop.renderString('{{{markup}}}', resultTemplate)
+            );
           });
-        });
-      },
-      function getStarted(answers) {
-        const {project} = answers as PromptsAnswers;
-        return `
-        To get started:
-        > cd ${project}
-        > npm run site:init (optional, sets up a new Netlify site)
-        > npm start
 
-        To share your site with the world:
-        > npm run site:deploy
-    
-        Happy hacking!
-        `;
-      },
-    ],
+          return `${page.results.templates.length} result template(s) generated`;
+        },
+        function installPackagesPrompt() {
+          return 'Installing packages...';
+        },
+        function installPackages(data) {
+          return new Promise((resolve, reject) => {
+            const {project} = data as PlopData;
+            const process = spawn(getPackageManager(), ['install'], {
+              stdio: 'ignore',
+              cwd: join(currentPath, project),
+            });
+
+            process.on('close', (code) => {
+              if (code === 0) {
+                resolve('Installation complete');
+              } else {
+                reject(`Installation exited with ${code}`);
```

Comment:
While at it, if you can bubble up the error or stderr, it'll help with maintenance and bugz hammering

Reply:
Already the case 😄  You can see this in stderr, plop does that for us